web/src/pages/Consumer/List.tsx

```diff
@@ -23,12 +23,15 @@ import { history, useIntl } from 'umi';
 import { PlusOutlined } from '@ant-design/icons';
 
 import { timestampToLocaleString } from '@/helpers';
+import { RawDataEditor } from '@/components/RawDataEditor';
```

Comment:
if only export one function, set it as the default usually.

Reply:
I think this is fine, if at a later stage to support other exports, so write changes will be less.

Reply:
Please consider the `React`